svc/AlertBannerService.ts

```diff
@@ -109,8 +111,42 @@ export class AlertBannerService extends HoistService {
         XH.localStorageService.set('xhAlertBanner.lastDismissed', Date.now());
     };
 
-    private isTargetedApp(clientApps: string[]): boolean {
-        return isEmpty(clientApps) || clientApps.includes(XH.clientAppCode);
+    /**
+     * Returns `true` if there are no limitations on application or if the
+     * current application is included in the defined limits.
+     * Returns `false` otherwise.
```

Comment:
Nit - we can leave off the "returns false otherwise" - implied.  TBH these first two private methods don't really need comments - they are pretty clear on their own.  Could amend the first to `isTargetedClientApp` for a bit of extra clarity.  A comment on the role method to indicate that user needs to have at least one of the roles might be helpful.